hrm-service/migrations/20200101000000-first_version.js

```diff
@@ -0,0 +1,183 @@
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.transaction(async transaction => {
+      const [result] = await queryInterface.sequelize.query(
+        `SELECT to_regclass('public."Contacts"') AS "tableExists"`,
+        {
+          transaction,
+        },
+      );
+      if (!result[0].tableExists) {
+        console.log('Attempting to build initial database schema.');
+        await queryInterface.sequelize.query(
+          `
+            CREATE TABLE public."AgeBrackets" (
```

Comment:
I would guess that the `AgeBrackets` and `Subcategories` tables can die, whenever it's a good time to remove them. They were from a very old initial database design that we did not use.

Reply:
I think they are already being removed in one of the early migrations